src/interactions/interaction.ts

```diff
@@ -11,8 +11,53 @@ module Plottable {
      */
     protected _componentToListenTo: Component;
 
-    public _anchor(component: Component) {
+    private _anchorCallback = (component: Component) => this._anchor(component);
```

Comment:
We'll probably want some sort of mechanism to detach the `Interaction` if the `Component` is `destroy()`ed, so we don't have loose references to the `Component` floating around and preventing garbage collection. However, that's an existing bug, so we can tackle it after the API refactor.